sdk/appcenter/src/main/java/com/microsoft/appcenter/channel/DefaultChannel.java

```diff
@@ -755,6 +793,7 @@ public void run() {
             mMaxParallelBatches = maxParallelBatches;
             mIngestion = ingestion;
             mListener = listener;
+            mPaused = false;
```

Comment:
nit: by convention we never initialize explicitly to default value, since false is already default

Reply:
I'm ok with changing it back if it's convention, but personally I can't remember default values for variables for a ton of different languages, so in my mind it becomes hidden logic.  I very much dislike this convention.

Reply:
Its null, 0 and false, there is nothing else actually.